scripts/docker/astrobee_base_bionic.Dockerfile

```diff
@@ -43,7 +43,8 @@ COPY ./scripts/setup/debians /setup/astrobee/debians
 
 RUN apt-get update \
   && /setup/astrobee/debians/build_install_debians.sh \
-  && rm -rf /var/lib/apt/lists/*
+  && rm -rf /var/lib/apt/lists/* \
+  && rm -rf /setup/astrobee/debians
```

Comment:
This actually reduced the size of the image by half!
I was getting insufficient space on device on the github CI run -.-